src/main/java/com/adyen/model/nexo/CapturedSignature.java

```diff
@@ -66,21 +70,21 @@ public void setAreaSize(AreaSize value) {
 
     /**
      * Gets the value of the signaturePoint property.
-     *
-     *
+     * <p>
+     * <p>
      * This accessor method returns a reference to the live list,
      * not a snapshot. Therefore any modification you make to the
      * returned list will be present inside the JAXB object.
      * This is why there is not a <CODE>set</CODE> method for the signaturePoint property.
-     *
-     *
+     * <p>
+     * <p>
      * For example, to add a new item, do as follows:
      * <pre>
      *    getSignaturePoint().add(newItem);
      * </pre>
-     *
-     *
-     *
+     * <p>
+     * <p>
+     * <p>
```

Comment:
I think it is by mistake.

Reply:
this was autogenerated from the jaxb tool, will update this in another ticket 👍